server/model/block.go

```diff
@@ -3,20 +3,27 @@ package model
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"io"
+	"regexp"
 	"strconv"
 	"unicode/utf8"
 
 	"github.com/mattermost/mattermost-plugin-boards/server/services/audit"
+	mmModel "github.com/mattermost/mattermost/server/public/model"
 )
 
 const (
+	MinIdLength         = 27
 	BlockTitleMaxBytes  = 65535                  // Maximum size of a TEXT column in MySQL
 	BlockTitleMaxRunes  = BlockTitleMaxBytes / 4 // Assume a worst-case representation
 	BlockFieldsMaxRunes = 800000
+	FileId              = "fileId"
```

Comment:
Great call, let's add `AttachmentId` as well, and I'd suggest to use a more specific name for the constant, a proposal could be:

```suggestion
	BlockFieldFileId       = "fileId"
	BlockFieldAttachmentId = "attachmentId"
```

Reply:
Updated.